### PR TITLE
PT-156004149: Update aehttp configuration

### PR DESCRIPTION
**Pull request by fabiankrol**

### CODE REVIEW

File: apps/aehttp/src/aehttp_app.erl [hunk 1]
```diff
@@ -121,14 +121,19 @@ get_and_parse_ip_address_from_config_or_env(CfgKey, App, EnvKey, Default) ->
     {ok, IpAddress} = inet:parse_address(binary_to_list(Config)),
     IpAddress.
 
-get_http_api_acceptors(_) -> ?INT_ACCEPTORS_POOLSIZE.
+get_http_api_acceptors(external) ->
+    aeu_env:user_config_or_env([<<"http">>, <<"external">>, <<"acceptors">>],
```
Comment (lucafavatella) [inline apps/aehttp/src/aehttp_app.erl]: I suggest enabling user to set this as per WS https://github.com/aeternity/epoch/blob/4795b2c820e1f44c23a92cb62ef5c85ec47cbd9e/apps/aeutils/priv/epoch_config_schema.json#L151

see also https://github.com/aeternity/epoch/blob/4795b2c820e1f44c23a92cb62ef5c85ec47cbd9e/apps/aeutils/test/data/epoch_full.yaml#L39

File: apps/aehttp/src/aehttp_app.erl [hunk 2]
```diff
@@ -121,14 +121,19 @@ get_and_parse_ip_address_from_config_or_env(CfgKey, App, EnvKey, Default) ->
...
+                               aehttp, [external, acceptors], ?INT_ACCEPTORS_POOLSIZE);
+get_http_api_acceptors(internal) ->
+    aeu_env:user_config_or_env([<<"http">>, <<"internal">>, <<"acceptors">>],
```
Comment (lucafavatella) [inline apps/aehttp/src/aehttp_app.erl]: I suggest enabling user to set this as per WS https://github.com/aeternity/epoch/blob/4795b2c820e1f44c23a92cb62ef5c85ec47cbd9e/apps/aeutils/priv/epoch_config_schema.json#L151

see also https://github.com/aeternity/epoch/blob/4795b2c820e1f44c23a92cb62ef5c85ec47cbd9e/apps/aeutils/test/data/epoch_full.yaml#L39

File: apps/aehttp/src/aehttp_app.erl (hunk 1 above)
Comment (fabiankrol) [reply apps/aehttp/src/aehttp_app.erl]: ✅

File: apps/aehttp/src/aehttp_app.erl (hunk 2 above)
Comment (fabiankrol) [reply apps/aehttp/src/aehttp_app.erl]: ✅

**Comment by fabiankrol** [comment]
@lucafavatella I'll add a line in release notes when the next release file is in place

File: apps/aehttp/src/aehttp_app.erl [hunk 3]
```diff
@@ -121,14 +121,19 @@ get_and_parse_ip_address_from_config_or_env(CfgKey, App, EnvKey, Default) ->
     {ok, IpAddress} = inet:parse_address(binary_to_list(Config)),
     IpAddress.
 
-get_http_api_acceptors(_) -> ?INT_ACCEPTORS_POOLSIZE.
+get_http_api_acceptors(external) ->
+    aeu_env:user_config_or_env([<<"http">>, <<"external">>, <<"acceptors">>],
+                               aehttp, [external, acceptors], ?INT_ACCEPTORS_POOLSIZE);
```
Comment (dincho) [inline apps/aehttp/src/aehttp_app.erl]: I guess using the same constant for internal/external pool size is intended.

**Comment by dincho** [comment]
I might want to remove the WIP label to not confuse reviewers. The label usually means - not ready, don't review yet.

File: apps/aehttp/src/aehttp_app.erl (hunk 3 above)
Comment (fabiankrol) [reply apps/aehttp/src/aehttp_app.erl]: Yes, that's correct.

**Comment by fabiankrol** [comment]
rebased

**Pull request merged**